### ISSUE TITLE
TransformStream ponyfill should only be used when necessary

### ISSUE THREAD

**Issue by bencmbrook** [transcend-io/conflux#52]
We may want to change Conflux to not overwrite the `TransformStream` variable if it's defined, e.g.

```js
import ponyfill from 'web-streams-polyfill/ponyfill';
const TransformStream = window.TransformStream || ponyfill.TransformStream
```

**Comment by jimmywarting** [comment]
Yep, notice it yesterday when i went back in for an effort to fix writing zip64 files.
have seen some unwanted changes (this being one of them)

I have written spec test for all kind of zip reading before i proceed fixing zip64.
going to make some test for writing too and then make a PR for tests

---

using native & polyfilled streams interfear with eachother. using `blob.stream()` or `response.body` don't work well right now b/c of this. thinking we should make a utility function that can read both asyncIterator and readable streams

(really wish the web stream polyfill extended the native stream)

**Comment by bencmbrook** [comment]
This should be resolved by #66 (orig: #53)
